tests/asyncio/test_retry_async.py

```diff
@@ -277,7 +277,7 @@ async def test___call___and_execute_success(self, sleep):
         sleep.assert_not_called()
 
     # Make uniform return half of its maximum, which is the calculated sleep time.
-    @mock.patch("random.uniform", autospec=True, side_effect=lambda m, n: n / 2.0)
```

Comment:
Why was this removed ?

Reply:
This is also a result of the change described in the item 8 of the PR description:

> Switch randomized delay calculation from delay being treated as expected value for randomized_delay to delay being treated as maximum value for randomized_delay (i.e. the new expected valud for randomized_delay is delay / 2). See the exponential_sleep_generator() method implementation for details. This is needed to make Python implementation of retries and polling exponential backoff consistent with the rest of GAPIC languages.